src/natcap/invest/sdr/sdr.py

```diff
@@ -213,24 +217,24 @@
     'sdr_bare_soil_path': 'sdr_bare_soil.tif',
     'sdr_path': 'sdr_factor.tif',
     'slope_path': 'slope.tif',
+    'stream_path': 'stream.tif',
```

Comment:
Was it desired to move stream path output to the intermediate folder? I feel like I remember prior conversations where it was requested stream path output go in the output folder...

Reply:
Oh, does the `stream_and_drainage_path` fill that need?

Reply:
Moved it back to the output folder. With the pour points thing it made more sense as an intermediate output, without that it belongs as a final output.